### PR TITLE
feat(TDI-45335): Bypass rate limit on docker hub

### PR DESCRIPTION
**Pull request by undx**
https://jira.talendforge.org/browse/TDI-45335

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/690/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/690/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/691/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/691/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/692/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/692/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/695/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/695/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/696/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/696/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/699/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/699/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/700/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/700/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/723/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/723/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/724/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/724/)

File: components/components-splunk/pom.xml [hunk 1]
```diff
@@ -170,7 +170,7 @@
             <plugin>
                 <groupId>io.fabric8</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
-                <version>${docker-maven-plugin.version}</version>
+                <version>0.26.0</version>
```
[the remark this answers is not in the document]
Comment (undx) [reply components/components-splunk/pom.xml:173]: We've some issues with that image when using parent's plugin version, that's why it's hardcoded and not using parent's one.

File: components/components-splunk/pom.xml [hunk 2]
```diff
@@ -183,7 +183,7 @@
                             <skip>${really.skip.tests}</skip>
                             <images>
                                 <image>
-                                    <name>dchmyga/splunk</name>
+                                    <name>artifactory.datapwn.com/docker-io-remote/dchmyga/splunk</name>
```
[the remark this answers is not in the document]
Comment (undx) [reply components/components-splunk/pom.xml:186]: Yes, it's customized not a bare splunk image.

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/729/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/729/)

**Pull request merged**